src/test/mochitest/browser_dbg-pause-on-next.js

```diff
@@ -0,0 +1,36 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Tests the breakpoints are hit in various situations.
+
+const expressionSelectors = {
+  plusIcon: ".watch-expressions-pane button.plus",
+  input: "input.input-expression"
+};
+
+async function addExpression(dbg, input) {
+  info("Adding an expression to evaluate");
+  findElementWithSelector(dbg, expressionSelectors.plusIcon).click();
+  findElementWithSelector(dbg, expressionSelectors.input).focus();
+  type(dbg, input);
+  pressKey(dbg, "Enter");
+}
+
+add_task(async function() {
+  const dbg = await initDebugger("doc-scripts.html");
+  const { selectors: { isPaused }, getState } = dbg;
+
+  log('Searching for simple3.js');
+  await selectSource(dbg, "simple3.js");
+
+  log('Press pause button');
+  clickElement(dbg, "pause");
+  await waitForDispatch(dbg, "BREAK_ON_NEXT");
+
+  await addExpression(dbg, "simple()");
```

Comment:
try `invokeInTab` here

Reply:
I tried replacing line 30 with `invokeInTab("simple");`

The test timed out . Here's the [log](https://git.io/vpsaQ). Any suggestions on why this may be happening?

Reply:
looks like it doesn't pause... that's weird :)